x/stake/client/cli/tx.go

```diff
@@ -165,7 +165,7 @@ func GetCmdDelegate(cdc *codec.Codec) *cobra.Command {
 				return utils.PrintUnsignedStdTx(txBldr, cliCtx, []sdk.Msg{msg})
 			}
 			// build and sign the transaction, then broadcast to Tendermint
-			return utils.SendTx(txBldr, cliCtx, []sdk.Msg{msg})
```

Comment:
Line 144: Please log to `os.Stderr`

Reply:
Why?

Reply:
Btw, all CLI contexts use STDOUT afaik.

Reply:
According to POSIX (http://pubs.opengroup.org/onlinepubs/009695399/) _standard error_ is _for writing diagnostic output_; _standard output_ is _for writing conventional output_. All CLI commands' logs should go to `/dev/stderr`.

Reply:
tbh, I'm not sure I fully understand. Why is traditional output such as JSON responses "diagnostic" output?

Reply:
Fair points -- I understand. But for another PR as well I think. Want to tackle that?

Reply:
Np, I'll open a non-prelaunch issue for that :+1: